doc/whats_new.rst

```diff
@@ -83,6 +87,12 @@ Bug fixes
       won't accept anymore ``min_samples_split=1`` as at least 2 samples
       are required to split a decision tree node. By `Arnaud Joly`_
 
+    - Resets the internal state of scaler before fitting (`#5416 <https://github.com/scikit-learn/scikit-learn/pull/5416>`_)
+
+    - Regression models now raises errors when targets are nan/inf (`#5431 <https://github.com/scikit-learn/scikit-learn/pull/5431>`_)
+
+    - Fixed serialization in the implementation of the pickle protocol (`#5560 <https://github.com/scikit-learn/scikit-learn/pull/5560>`_)
+
```

Comment:
Please add bug fixes:

    - Fixed a test failure when using a development install of scipy. The error was:
      ValueError: invalid literal for int() with base 10: 'dev0+1072f2e'
      by `CJ Carey`_